.github/workflows/release.yaml

```diff
@@ -9,50 +9,59 @@ on:
 jobs:
   release-rulesfile:
     runs-on: ubuntu-latest
-    env:
-      AWS_S3_BUCKET: falco-distribution
-      AWS_S3_PREFIX: rules
-      AWS_S3_REGION: eu-west-1
 
-    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
     permissions:
+      # These permissions are needed to interact with GitHub's OIDC Token endpoint.
       id-token: write
       contents: read
       packages: write
 
+    env:
+      RULES_FILE: falco_rules.yaml
+
+      OCI_REGISTRY: ghcr.io
+      ARTIFACT_NAME: falco-rules
+
     steps:
+
+      # Get rules repository
       - name: Checkout Rules
         uses: actions/checkout@v3
 
+      # Get registry artifact tool
       - name: Setup Golang
         uses: actions/setup-go@v3
         with:
-          go-version: '^1.19'
-
+          go-version-file: build/registry/go.mod
       - name: Build registry artifact tool
         working-directory: build/registry
         run: go build -o rules-registry ./...
-
       - name: Get lowercase OCI repo prefix
         run: |
           echo "OCI_REPO_PREFIX=ghcr.io/${GITHUB_REPOSITORY,,}" >>${GITHUB_ENV}
-
       - name: Upload OCI artifacts to GitHub packages
+        id: oci_build
         env:
           REGISTRY_USER: ${{ github.repository_owner }}
           REGISTRY_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_REPO_URL: ${{ github.server_url }}/${{ github.repository }}.git
 
         # uses OCI_REPO_PREFIX environment variable
-        run: build/registry/rules-registry push-to-oci registry.yaml ${{ github.ref_name }}
+        run: >-
+          echo "::set-output name=ARTIFACT_DIGEST::$(
+            build/registry/rules-registry push-to-oci registry.yaml ${{ github.ref_name }}
+          )"
```

Comment:
I believe `::set-output` is deprecated and we should use the GITHUB variables instead: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Good catch @LucaGuerra thanks!

Reply:
@LucaGuerra fixed it, PTAL, thanks